src/SFA.DAS.ApplyService.InternalApi/Controllers/ReviewController.cs

```diff
@@ -27,27 +26,6 @@ public ReviewController(IMediator mediator)
             _mediator = mediator;
         }
 
-        [HttpGet("Review/OpenApplications")]
-        public async Task<ActionResult> OpenApplications(int sequenceId = 1)
-        {
-            var applications = await _mediator.Send(new OpenApplicationsRequest(sequenceId));
-            return Ok(applications);
-        }
-
-        [HttpGet("Review/FeedbackAddedApplications")]
-        public async Task<ActionResult> FeedbackAddedApplications()
-        {
-            var applications = await _mediator.Send(new FeedbackAddedApplicationsRequest());
-            return Ok(applications);
-        }
-
-        [HttpGet("Review/ClosedApplications")]
-        public async Task<ActionResult> ClosedApplications()
-        {
-            var applications = await _mediator.Send(new ClosedApplicationsRequest());
-            return Ok(applications);
-        }
```

Comment:
These will be required later

Reply:
They've been moved into the ApplicationController, so that everything is in the same place